meta-client/src/main/java/org/apache/helix/metaclient/impl/zk/ZkMetaClient.java

```diff
@@ -90,12 +90,17 @@ public void create(String key, Object data, MetaClientInterface.EntryMode mode)
 
   @Override
   public void createWithTTL(String key, T data, long ttl) {
-    throw new UnsupportedOperationException("TTL nodes aren't yet supported.");
+    try{
+      _zkClient.createPersistentWithTTL(key, data, ttl);
+    } catch (ZkException e) {
+      throw translateZkExceptionToMetaclientException(e);
+    }
   }
 
+  //TODO: Implement logic for renewTTL once expiry time can be verified through testing.
   @Override
   public void renewTTLNode(String key) {
-    throw new UnsupportedOperationException("TTL nodes aren't yet supported.");
+    throw new NotImplementedException("Renew TTL node is not available yet.");
```

Comment:
Thanks for making the change. I would still suggest implementing renewTTL and add TODO for TTL node check later when possible.

Reply:
Sounds good, added the changes